wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Insights > Posting Activity

Fixes #10591 

This adds the `Posting Activity` card to the Insights table. It's made up of these components:
- `PostingActivityDay` - Shows the square representing a day, color coded per the post count.
- `PostingActivityMonth` - Shows the month grid, built with `PostingActivityDay`s.
  - A bit of new magic - most months span 5 weeks, however there are a few that span 6. December 2018 is an example. There is now a 6th column in the month grid that will show/hide depending on if it's used or not.
- `PostingActivityLegend` - Shows the `Fewer Posts ..... More Posts` view.
- `PostingActivityCell` - Shows the Posting Activity in the Insights table, which includes:
  - The last three months.
  - The `PostingActivityLegend`.
  - `View more`, which displays a temporary alert.

To test:
- Go to Stats > Insights.
- Verify Posting Activity appears.
- Revel in the beauty.
<img width="464" alt="posting_activity" src="https://user-images.githubusercontent.com/1816888/49601599-7ec89600-f943-11e8-8fd4-295420a8e049.png">
